wdio.conf.js

```diff
@@ -155,7 +158,12 @@ exports.config = {
   sauceConnect: true,
 
   sauceConnectOpts: {
-    tunnelIdentifier: os.hostname()
+    tunnelIdentifier: os.hostname(),
+    logfile: './sauce-connect.log',
```

Comment:
should we keep this?

Reply:
No, we shouldn't, but we can use the provided infos when its necessary. Maybe it's overkill a bit.